prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-actions/adjudicator-actions.component.ts

```diff
@@ -219,4 +216,8 @@ export class AdjudicatorActionsComponent implements OnInit, OnChanges {
         this.assignToa.emit({ enrolleeId: this.enrollee.id, agreementType })
       );
   }
+
+  public canChangeToA() {
+    return this.enrollee.appliedDate !== null && this.enrollee.currentStatusCode === this.EnrolmentStatus.EDITABLE;
```

Comment:
Should be `approvedDate`?  E.g. this enrollee hasn't been approved yet but was made Editable, and now admin has "Change TOA" option

![image](https://github.com/user-attachments/assets/d2f59f3a-e468-4a27-8879-7c9386c763be)

Reply:
Good catch. It is fixed.